src/Umbraco.Web.BackOffice/Umbraco.Web.BackOffice.csproj

```diff
@@ -32,7 +32,7 @@
     <ItemGroup>
         <ProjectReference Include="..\Umbraco.Configuration\Umbraco.Configuration.csproj" />
         <ProjectReference Include="..\Umbraco.Core\Umbraco.Core.csproj" />
-        <ProjectReference Include="..\Umbraco.Examine.Lucene\Umbraco.Examine.Lucene.csproj" />
+        <ProjectReference Include="..\Umbraco.Examine.Lucene\Umbraco.Examine.Lucene.csproj" Condition="'$(OS)' == 'Windows_NT'" />
```

Comment:
neat, i didn't know about the OS conditionals

Reply:
Me2, I never needed those before now 😄